Blog: add interview with Matthew Mols/pgEdge (#399)
cloudnative-pg/cloudnative-pg.github.io: content/blog/matthew-mols-pgedge/index.md

---
title: "Sticking with Open Source: pgEdge and CloudNativePG"
date: 2025-12-22
draft: false
image:
    url: pgedge_cloudnativepg.jpg
    attribution:
authors:
 - fdrees
tags:
 - helm
 - ImageCatalog
 - CNCF
 - kubernetes
 - postgresql
 - open-source
summary: "We talked to Matthew Mols, Sr. Director of Engineering at pgEdge, about how CloudNativePG enables them to meet the requirements of their customers using just open source." 
---

[Matthew Mols](https://www.linkedin.com/in/mmols/) is the Sr. Director of Engineering at pgEdge, a team of engineers and entrepreneurs on a mission to make it easy to build, deploy and manage enterprise grade applications at scale on Postgres. 

Recently pgEdge announced their [CloudNativePG intergration](https://www.postgresql.org/about/news/pgedge-announces-cloudnativepg-integration-simplifying-postgres-on-kubernetes-3166/) and them joining the Cloud Native Computing Foundation (CNCF). 

We had a chance to talk to Matt about their use of CloudNativePG.

## Why CloudNativePG works for pgEdge

> Matt loves learning about customers' challenges with using Postgres, and thinking about how they can build or suggest tools and approaches to make their lives easier. "I'm continuously surprised by the different ways folks are leveraging Postgres in their businesses!"

Reviewer: Small suggestions. BTW even if Matt did say "continuously", I think he meant "continually".

```suggestion
Matt loves learning about customers' challenges with Postgres, and thinking about how they can build or suggest tools and approaches to make their lives easier. "I'm continually surprised by the different ways folks are leveraging Postgres in their businesses!"
```

> Matt's role is focused on developing tools that enable pgEdge's customers to effectively deploy Postgres, whether that’s on Kubernetes, VMs, Bare Metal, or in the Cloud. A lot of this work is centered around making it easier to use tools together to meet the requirements of different shapes of customers. 

Reviewer: Capitalization of bare metal sounds strange, but I may be wrong.

```suggestion
Matt's role is focused on developing tools that enable pgEdge's customers to effectively deploy Postgres, whether that’s on Kubernetes, VMs, bare metal, or in the Cloud. A lot of this work is centered around making it easier to use tools together to meet the requirements of different kinds of customers. 
```

"We are fully committed to open source and tend to utilize a combination of open source extensions and tools that we've developed and released, like Spock for multi-master (active-active) logical replication, combined with stable community tools like CloudNativePG." 

pgEdge utilizes CloudNativePG in our their [Helm chart](https://docs.pgedge.com/pgedge-containers/), which allows for users to deploy active-active databases into multiple Kubernetes clusters, and keep them in sync. 

## Getting started with CloudNativePG

> Before CloudNativePG, Matt and his team leveraged other operators, and a mix of custom Helm charts that leveraged Kubernetes primitives to deploy Postgres instances. CloudNativePG's popularity and stability, and its acceptance into the CNCF confirmed that it was the right choice to switch to as the default.

Reviewer: Two "leveraged" very close.

```suggestion
Before CloudNativePG, Matt and his team used other operators, and a mix of custom Helm charts that leveraged Kubernetes primitives to deploy Postgres instances. CloudNativePG's popularity and stability, and its acceptance into the CNCF, confirmed that it was the right choice to switch to as the default.
```
